tests/test_catalog.py

```diff
@@ -1,10 +1,12 @@
+from conftest import data_dir
+
 from btk.catalog import CatsimCatalog
 from btk.catalog import CosmosCatalog
 
-CATALOG_PATH = "data/sample_input_catalog.fits"
+CATALOG_PATH = (data_dir / "sample_input_catalog.fits").as_posix()
 COSMOS_CATALOG_PATHS = [
-    "data/cosmos/real_galaxy_catalog_23.5_example.fits",
-    "data/cosmos/real_galaxy_catalog_23.5_example_fits.fits",
+    (data_dir / "cosmos/real_galaxy_catalog_23.5_example.fits").as_posix(),
+    (data_dir / "cosmos/real_galaxy_catalog_23.5_example_fits.fits").as_posix(),
```

Comment:
So about that change you made..the goal of using Path was to be able to handle both posix (Unix like) and Windows systems. By restricting it, it loses its interest. Could you put the Traceback of the error you had before changing it please ?

Reply:
@ismael-mendoza could you try with calling `str()` on the Path rather than using `.as_posix()` ?

Reply:
oops you are right, I thought `as_posix()` was equivalent to `str()` but I guess not.

Reply:
I changed the calls to `str()` and the tests still pass